conda env example on IMC/IF napping (#14)
BodenmillerGroup/napari-imcreg: README.md

To start `napping` from command line:

    napping
    

> ## Environment for IMC and IF mapping

Reviewer: ```suggestion
## Environment setup example: mapping IMC and slidescanner images
```

> Certain dependencies need to be installed for aligning imaging mass cytometry (IMC) and brightfield/immunofluorescence images:

Reviewer: ```suggestion
As napari is under active development, the use of virtual environments is strongly encouraged. For example, to setup an environment for aligning Imaging Mass Cytometry (IMC, .mcd/.txt) and brightfield/immunofluorescence slidescanner (.czi) images:
```

>     conda create -n napping_imc python

Reviewer: ```suggestion
    conda create -n napping python
```

>     conda activate napping_imc

Reviewer: ```suggestion
    conda activate napping
```
